Homework/homework3/test_task1.py

```diff
@@ -0,0 +1,19 @@
+from unittest.mock import Mock
+
+from homework3.task_1 import cache
+
+import pytest
+
+
+@pytest.mark.parametrize(
+    "times,calls,expected_result", [(1, 4, 2), (2, 4, 2), (3, 4, 1), (3, 5, 2)]
```

Comment:
What the difference between tests?  Does the next one bring additional value?

Reply:
Just to ensure that its work well with different values of times and different number of calls of function (for not calling function manually).